addons/io_hubs_addon/components/definitions/loop_animation.py

```diff
@@ -35,18 +35,24 @@ def draw_item(self, context, layout, data, item, icon, active_data, active_propn
 class AddTrackOperator(Operator):
     bl_idname = "hubs_loop_animation.add_track"
     bl_label = "Add Track"
+    bl_options = {'REGISTER', 'UNDO'}
 
     track_name: StringProperty(
         name="Track Name", description="Track Name", default="")
 
+    panel_type: StringProperty(name="panel_type")
+
     def execute(self, context):
+        panel_type = PanelType(self.panel_type)
```

Comment:
Once the `context_pointer_set` has been set for the layout can't we use the panel here same as we are doing for `RemoveTrackOperator` and get rid of the property?

Reply:
Yes, that's possible, and I did consider it, but I think having the arguments is clearer than using `context_pointer_set`.  The only reason I used it for the `RemoveTrackOperator` was because I needed to access it in the `poll` function and you can't access the arguments from there.  That said, we are using it for the `RemoveTrackOperator`, so I don't have a really strong feeling about this and I'm fine with changing it to use `context_pointer_set` if you want.

Reply:
Ok, I see. I was just curious why but I'm ok with leaving it like it is.